src/main/resources/anniv5Resources/localization/eng/Cardstrings.json

```diff
@@ -19,5 +19,9 @@
     "NAME": "Conjure Pack",
     "DESCRIPTION": "Choose 1 of !M! Packs available to your run. NL At the start of the next 5 turns, add 2 of its cards to your hand.",
     "UPGRADE_DESCRIPTION": "Innate. NL Choose 1 of !M! Packs available to your run. NL At the start of the next 5 turns, add 2 of its cards to your hand."
+  },
+  "${ModID}:Vexed": {
+    "NAME": "Vexed",
+    "DESCRIPTION": "Unplayable. NL Cannot be removed from your deck. NL While in hand, Exhaust this when you play a *Foil card."
```

Comment:
I took the "While in hand" part of the wording from Pain, which is one of the only base game cards with an effect like this (Normality also uses "While in hand" though that effect is constant rather than triggered).